SPECS/edk2/edk2.spec

```diff
@@ -785,6 +791,12 @@ done
 /boot/efi/HvLoader.efi
 
 %changelog
+* Thu Feb 27 2025 Archana Choudhary <archana1@microsoft.com> - 20240524git3e722403cd16-5
+- Add patch for CVE-2023-6129, CVE-2024-2511, CVE-2024-4603
+
+* Wed Sep 25 2024 Cameron Baird <cameronbaird@microsoft.com> - 20240524git3e722403cd16-4
```

Comment:
Why did we jump two versions/include this changelog entry?

Reply:
Please rebase this change on the following commit: https://github.com/microsoft/azurelinux/commit/c45bed25df4e35298bcaba68f14322770e851760 

And while doing that bump the version number for the edk2-hvloader-signed package.

Reply:
Base branch change led to that. Its removed. I cherry-picked the mentioned commit.